packages/pages/src/components/image/image.tsx

```diff
@@ -0,0 +1,181 @@
+import * as React from "react";
+import { useEffect, useRef, useState } from "react";
+import { ImageProps, ImageLayout } from "./types";
+
+/**
+ * A component that renders an image. Here is an example of using the component with simple and
+ * complex image fields from Knowledge Graph:
+ * ```
+ * import { Image } from "@yext/pages/components";
+ *
+ * const template = ({ document }) => {
+ *   return (
+ *     <div>
+ *       <Image image={document.logo.image} />
```

Comment:
Did we consider making the prop just `document.logo` to simplify things a bit?

Reply:
If we want to make this change, the `ImageType` will need to be `type ImageType = {image: {height: number...};};` with the extra `image` layer.

Reply:
I agree with Max that we should make it simpler to use.